content/java/concepts/calendar/terms/compareTo/compareTo.md

```diff
@@ -0,0 +1,68 @@
+    ---
+    Title: '.compareTo()'
+    Description: 'Compares an Object of Calendar passed as parameter with this Calendar Object'
+    Subjects:
+    - 'Code Foundations'
+    - 'Computer Science'
+    Tags:
+    - 'Calendar'
+    - 'Date'
+    - 'Methods'
+    CatalogContent:
+    - 'learn-java'
+    - 'paths/computer-science'
+    ---
+
+    The **`.compareTo()`** method of the `Calendar` class is used to compare the time values or the millisecond offsets of this Calendar object with the passed Calendar object.
+
+    ## Syntax
+
+    ```pseudo
+    result = myCalendar.compareTo()
+    ```
+
+    The method returns an `Integer` value and can return any one of the following:
+
+    The method returns `0` if the passed argument is equal to this `Calendar` object.
+    The method returns `1` if the time of this Calendar object is more than the passed object.
+    The method returns `-1` if the time of this Calendar object is less than the passed object.
+
+    ## Example
+
+    This example illustrate the working of `compareTo()` Method of Calendar class.
+
+    ```java
+    import java.util.*;
+    
+    public class CalendarClassDemo {
+        public static void main(String args[])
+        {
+            // Creating a calendar object
+            Calendar calndr1
+                = Calendar.getInstance();
+    
+            // Creating another calendar object
+            Calendar calndr2
+                = new GregorianCalendar(2018, 12, 2);
+    
+            // Comparing the time
+            int val = calndr1.compareTo(calndr2);
+    
+            // Displaying the result of comparison
+            System.out.println("First" + " comparison result is: " + val);
+    
+            // Comparing the time
+            val = calndr2.compareTo(calndr1);
+    
+            // Displaying the result of comparison
+            System.out.println("Second"+ " comparison result is: "+ val);
+        }
+    }
+    ```
```

Comment:
````suggestion
```java
import java.util.*;
    
public class CalendarClassDemo {
    public static void main(String args[])
    {
        // Creating a calendar object
        Calendar calndr1
            = Calendar.getInstance();

        // Creating another calendar object
        Calendar calndr2
            = new GregorianCalendar(2018, 12, 2);

        // Comparing the time
        int val = calndr1.compareTo(calndr2);

        // Displaying the result of comparison
        System.out.println("First" + " comparison result is: " + val);

        // Comparing the time
        val = calndr2.compareTo(calndr1);

        // Displaying the result of comparison
        System.out.println("Second"+ " comparison result is: "+ val);
    }
}
```
````